types/interfaces.ts

```diff
@@ -2,6 +2,7 @@ import { Date, Types } from 'mongoose'
 import Logger from 'bunyan'
 import { UserDoc } from '../server/models/User'
 import { XYPosition } from 'react-flow-renderer'
+import { boolean } from 'yargs'
```

Comment:
This shouldn't be an import.

Reply:
`boolean` is a basic type that _should_ exist by default.

Reply:
This looks like an automatic import from my IDE! Will remove.